pkg/server/server.go

```diff
@@ -181,6 +183,21 @@ func NewServer(o *Options) (*Server, error) {
 		return nil, err
 	}
 
+	if o.podIptables != "" {
+		// cleanup current pod iptables directory if it exists
+		if _, err := os.Stat(o.podIptables); err == nil || !os.IsNotExist(err) {
+			err = os.RemoveAll(o.podIptables)
```

Comment:
Hey Tomo, just making sure I understand this one...

If the stat comes back that the file exists... it removes all files in that path? And this just occurs when the server is started? I think that's probably OK. Just making sure I understand.

Do we need to check that the path is actually a directory and not a file itself? E.g. I think in this case if you put `/etc/system-release` it would delete the file @ `/etc/system-release` and then create it as a directory.

Reply:
Ok, I think I see now that this `podIptables` is generated with the `--pod-iptables=*` + the pod UID. Now I get it. I think this is good. That should be deleted. I was worried we'd lose history if the server crashed.